source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,401 @@
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
 
-Multiple paths should be used to make the requests. When possible, 50% of items requested should be via browsing the platform and 50% via searching the platform.
+**Option 5**: Platform does not include any databases.
 
-If browsing to items or accessing items via searching is not possible, then 100% of requested items can be accessed via the only available option.
+The auditor MUST run 100 searches on the platform.
 
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
 
-Audit Test D1-3: Total_Item_Requests - 30-second filters
-........................................................
+E.2.2.2 Platform Investigations and Requests
+''''''''''''''''''''''''''''''''''''''''''''
 
-The audit test consists of making an Item_Request twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second Total_Item_Request must be recorded. If the two clicks occur with more than 30 seconds between them, then 2 Total_Item_Requests must be counted.The auditor must carry out a total of 30 tests, and each test will consist of 2 requests. There are 2 types of tests that must be carried out:
+Where possible, 50% of items requested SHOULD be via browsing the platform and 50% via searching the platform. If either browsing or searching is not possible, all items can be requested via the only available option.
 
-* “Inside” tests (The 2 requests are made to the same item, and the second request is made within 30 seconds of the first).
-* “Outside” tests (The 2 requests are made to the same item, and the second request is made more than 30 seconds after the first).
+**Option 1**: Platforms with mixtures of Data_Types including Data_Type=Book and/or Reference_Work.
 
-The auditor must carry out 15 **inside** tests.
+The auditor MUST make a total of 100 requests
 
-This must result in:
+* 50 requests against different items with Data_Type other than Book and/or Reference_Work.
+* 10 requests against different items within each of 5 books and/or reference works, for 50 total requests.
 
-* 15 Total_Item_Requests being reported in the DR_D1 Standard View.
+The auditor activity MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 5 each of Unique_Title_Investigations and Unique_Title_Requests (1 per book and/or reference work).
 
-The auditor must carry out 15 **outside** tests.
+If a platform only delivers book and/or reference work content as a single download (i.e. without Book_Segments and/or Reference_Items), then each request SHOULD be for a separate book and/or reference work. In this case the auditor activity MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 50 each of Unique_Title_Investigations and Unique_Title_Requests (1 per book and/or reference work).
```

Comment:
This is only correct when the Book_Segments/Reference_Items cannot be identified, otherwise the Total/Unique_Item counts are the sum of the Book_Segements/Reference_Items.